docs/azure/DeviceCredentials.rst

```diff
@@ -1,77 +1,90 @@
 .. _azure-device-credentials:
 
-Device Credentials
+Device credentials
 ##################
 
-On Azure devices connect to IoT Hub using `TLS version 1.2 <https://docs.microsoft.com/en-us/azure/iot-fundamentals/iot-security-deployment>`_ and Elliptic Curve Cryptography (ECC) based certificates.
+On Azure devices, connect to IoT Hub using `TLS version 1.2 <https://docs.microsoft.com/en-us/azure/iot-fundamentals/iot-security-deployment>`_ and Elliptic Curve Cryptography (ECC) based certificates.
 
 Azure supports offline generation of device certificates through `Enrollment Groups <https://docs.microsoft.com/bs-latn-ba/azure/iot-dps/quick-enroll-device-x509-node>`_.
 
 Device certificates are signed using an intermediate CA.
 This allows to keep the CA root certificate secure and create an intermediate CA certificate to be used during device provisioning.
-This way the CA root certificate private key will never be transmitted to insecure locations (e.g. a third-party factory).
+In this case, CA root certificate private key will never be transmitted to insecure locations (for example, a third-party factory).
```

Comment:
```suggestion
Because of that, CA root certificate private key will never be transmitted to insecure locations (for example, a third-party factory).
```

This is not referring to different cases/scenarios, it's a reference to the decision of "using an intermediate CA."